cdmsparkevents/eventloop.py

```diff
@@ -272,9 +272,16 @@ def _run_integration_test(self, val: dict[str, Any]):
 
     def _run_importer(self, importer_module: str, app_name: str, job_info: dict[str, Any]):
         mod = importlib.import_module(importer_module)
-        def get_spark(executor_cores: int = 1) -> SparkSession:
-            return spark_session(self._cfg, app_name, executor_cores=executor_cores)
-        mod.run_import(get_spark, job_info)
+        sparkcapture = []
+        def get_spark(*, executor_cores: int = 1) -> SparkSession:
```

Comment:
I feel like doing a spark context manager looks cleaner here. But it works.

Reply:
I don't think that'll work here because the spark session is created in `get_spark`, will it?

Reply:
hmmm it won't?

I was thinking something like this. Maybe it won't work. 
```    
with spark_context(cfg, app_name, executor_cores) as spark:
       def get_spark(*, executor_cores: int = executor_cores) -> SparkSession:
            return spark
       mod.run_import(get_spark, job_info)

```

Reply:
In that case is looks like the user-supplied executor_cores argument when the get_spark method is called in the run_import method is ignored